cumulus_library/apis/loinc.py

```diff
@@ -31,6 +31,8 @@ def __init__(self, *, user: str | None = None, password: str | None = None):
 
         self.session = requests.Session()
         self.session.auth = requests.auth.HTTPBasicAuth(user, password)
+        self.cache_dir = base_utils.get_user_cache_dir() / "loinc"
+        self.download_dir = self.cache_dir / "downloads"
```

Comment:
On this and the other api module - there's not great support for 'required instance variables that must be populated defined in an abstract base class', and so I didn't go down one of several workarounds now, but I think my vibe is that each API module should provide three variables: a session with auth configured, a cache dir, and a download path inside that cache dir. If we add a third API I'd revisit this.